packages/frontend/src/components/UserModification/UserModification.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import { TextField, Button, FormControlLabel, Checkbox } from "@mui/material";
+import { useAppDispatch } from "../../app/hooks";
+import { asyncUserModification } from "../../app/trunks";
+import { ICustomer } from "../helpers/interfaces";
+import { modificationSchema } from "../helpers/validationSchemas";
+import "./UserModification.scss";
+
+interface IUserModificationProps {
+  user: ICustomer;
+  setChangeOpened: React.Dispatch<React.SetStateAction<boolean>>;
+  setDeletedCustomerId: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const UserModification: React.FC<IUserModificationProps> = ({
```

Comment:
I think if we rename this component (and filenames) to "EditUserForm" it will be more clear

Reply:
done